src/cryptography/utils.py

```diff
@@ -16,6 +16,10 @@
 # cycle ends.
 PersistentlyDeprecated = DeprecationWarning
 DeprecatedIn19 = DeprecationWarning
+# Asymmetric signature and verification contexts were deprecated in 2.0
+# However they are widely used and we will obey a longer than normal
+# deprecation cycle for them.
+PersistentlyDeprecatedIn20 = PendingDeprecationWarning
```

Comment:
Is there a reason not to use the same `PersistentlyDeprecated` class, or is your expectation that we'll migrate to them once this switches from `Pending` to regular?